app/components/public/featured-speaker-list.js

```diff
@@ -1,5 +1,19 @@
 import classic from 'ember-classic-decorator';
 import Component from '@ember/component';
+import { tracked } from '@glimmer/tracking';
+import { hasSpeakers } from 'open-event-frontend/utils/event';
 
 @classic
-export default class FeaturedSpeakerList extends Component {}
+export default class FeaturedSpeakerList extends Component {
+
+    @tracked allSpeakers = null;
+
+    async didInsertElement() {
+      super.didInsertElement(...arguments);
+      this.countSpeakers();
+    }
+
+    async countSpeakers() {
+      this.allSpeakers = this.allSpeakers ?? await hasSpeakers(this.loader, this.event, true);
```

Comment:
No need to add one more query as discussed in meeting several times

Reply:
it's not working that way, I have tried that.

Reply:
But we have to use `hasSpeaker` somewhere ? without adding it no where how it will get value.

Reply:
It is already called in side menu, move it to public controller